src/pages/settled.tsx

```diff
@@ -3,9 +3,10 @@ import { useOracleDataContext } from "@/hooks";
 
 export default function Propose() {
   const { settled } = useOracleDataContext();
+
   return (
     <Layout>
-      <Table rows={settled} isLoading={false} page="propose" />
+      <Table rows={settled} isLoading={settled.length === 0} page="settled" />
```

Comment:
we might need to add specific loading flag, as we arent gauranteed to always ahve values for these pages. what do u think?